projects/packages/forms/src/dashboard/class-dashboard.php

```diff
@@ -111,7 +111,7 @@ public function add_admin_submenu() {
 				'',
 				__( 'Form Responses', 'jetpack-forms' ),
 				_x( 'Feedback', 'post type name shown in menu', 'jetpack-forms' ),
-				'read',
+				'edit_pages',
 				'jetpack-forms',
 				array( $this, 'render_dashboard' )
 			);
```

Comment:
Just like above, this update matches the behavior of the list page view, in the package and in the legacy module:
https://github.com/Automattic/jetpack/blob/ca5ff93ae6d54977a25434dd8afb2f62ab9a45ed/projects/packages/forms/src/contact-form/class-contact-form-plugin.php#L500-L518
https://github.com/Automattic/jetpack/blob/653476e520d896d175fe995e8f190d8403795a83/projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php#L743-L761